ui/src/translations/fr.json

```diff
@@ -179,10 +179,7 @@
   "start": "Début",
   "no_data_available_for_metrics": "Aucune donnée disponible pour afficher les Métriques",
   "global_health_explanation": "L'indicateur de santé generale frounit une information sur le status de votre Platform pour une periode donnée.\nLe statut des Volumes et des Noeuds, celui du Reseau et des Services sont supervisés.\nUn cercle de couleur verte signifie que la Platform est en bonne santé.\nUn cercle de couleur orange signifie que la Platform est degradé mais pas en danger.\nUn cercle de couleur verte signifie que la Platform est degradée et en danger.\nSurvolez les sgements du graphique pour plus de details.",
```

Comment:
A lot of typos here: "frounit" => "fournit", "status" => "statut" and many more

Reply:
Can you open a JIRA issue to track those typos? This line was not changed in this PR, so let's not ask YanJin to fix it all here :wink: